app-vite/package.json

```diff
@@ -72,7 +72,6 @@
     "minimist": "^1.2.6",
     "open": "^8.4.0",
     "ouch": "^2.0.0",
-    "postcss": "^8.4.4",
```

Comment:
Isn't this removal a breaking change? Can you explain why you needed to move it to the project `package.json`?

Reply:
[`4b75327` (#13615)](https://github.com/quasarframework/quasar/pull/13615/commits/4b7532724a52e84a62fe7eb34749b7435350d45b)

> chore(app-vite): Move postcss to devland
We are not directly using it in app-vite, so we can move it to devland.
This helps to avoid missing peer deep errors/warnings from autoprefixer,
which is installed on the default template, and is basically required.
Existing apps will not be affected because Vite also ships postcss as
a dependency.

Reply:
Sorry, didn't think about checking the commits, that's clear now